apiv2/schema/schema.yaml

```diff
@@ -816,7 +809,7 @@ classes:
       plural:
         tag: plural
         value: Annotations
-    description: Metadata about an annotation for a run
```

Comment:
I think this table is missing the `method_links` json field.

Reply:
Added!